src/pysdk/grvt_raw_signing.py

```diff
@@ -100,7 +110,18 @@ def sign_order(
     domain_data = get_EIP712_domain_data(config.env)
     signature = encode_typed_data(domain_data, EIP712_ORDER_MESSAGE_TYPE, message_data)
     signed_message = account.sign_message(signature)
-    order.signature.r = "0x" + hex(signed_message.r)[2:].zfill(64)
+
+    # Normalize the s value
+    normalized_s = normalize_s(signed_message.s)
+    order.signature.s = "0x" + normalized_s.to_bytes(32, byteorder='big').hex()
+
+    # Handle the v value
+    # Ensure v is either 27 or 28
+    v = signed_message.v
+    if v not in (27, 28):
+        v = v + 27 if v < 27 else v
+    order.signature.v = v
+
     order.signature.s = "0x" + hex(signed_message.s)[2:].zfill(64)
```

Comment:
@minhbsq this sets s to the value before normalisation. Take the value from order.signature.s?

Reply:
my bad, should have remove the old statements to assign s & v to order.signature. Fixed now